### PR TITLE
Add validate contact form

### PR DESCRIPTION
**Pull request by MostafaShahat2021**
### ⭐in this branch we:
- ⭐When the user submits the form, we check if the email is in lowercase.
    - If the validation is OK✅, the form is sent.
    - If the validation is not OK❌, we show an error message to the user informing him that the form is not sent.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -178,13 +178,14 @@ <h3 class="work-titl work-titl-form">CONTACT US</h3>
...
             <ul>
                 <li>
                     <input type="text" name="user_name" id="user_name" placeholder="Your Name" minlength="5"
                         maxlength="30" required>
                 </li>
                 <li><input type="email" name="user_mail" id="user_mail" placeholder="jeremy@gmail.com" required></li>
+                <li class="msg">Message</li>
                 <li>
                     <textarea name="user_text" id="user_text" cols="46" rows="10" placeholder="Enter your message here"
                         minlength="10" maxlength="500" required></textarea>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please kindly ensure that the error message appears close to the button as per requirements.

Comment (AnselemOdims) [inline index.html]: - [ ] Please kindly ensure that the error message appears close to the button as per requirements.

**Review by AnselemOdims — APPROVED** [review]
### Hi @MostafaShahat2021 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (MostafaShahat2021) [reply index.html]: @AnselemOdims thank you so much for your efforts in reviewing my code

**Review by ghost — APPROVED** [review]
Hi  @MostafaShahat2021 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)


## Highlights

✅ No linter error
✅ README is okay
✅ Validation works 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MostafaShahat2021** [comment]
@frank1738 Thank you so much for your efforts in reviewing my code.

**Pull request merged**